client/src/modules/privacy-policy/components/PrivacyPolicy.en.md

```diff
@@ -0,0 +1,233 @@
+
+- **To enforce our terms, conditions and policies.**
+
+- **To respond to legal requests and prevent harm.** If we receive a subpoena or other legal request, we may need to inspect the data we hold to determine how to respond.
+
+- **For other Business Purposes.** We may use your information for other Business Purposes, such as data analysis, identifying usage trends, determining the effectiveness of our promotional campaigns and to evaluate and improve our Sites or Apps, products, services, marketing and your experience.
+
+### [3. WILL YOUR INFORMATION BE SHARED WITH ANYONE?](#3-will-your-information-be-shared-with-anyone)
+
+**_In Short:_** We only share information with your consent, to comply with laws, to protect your rights, or to fulfill business obligations.
+
+We may process or share data based on the following legal basis:
+
+- **Consent:** We may process your data if you have given us specific consent to use your personal information in a specific purpose.
+
+- **Legitimate Interests:** We may process your data when it is reasonably necessary to achieve our legitimate business interests.
+
+- **Performance of a Contract:** Where we have entered into a contract with you, we may process your personal information to fulfill the terms of our contract.
+
+- **Legal Obligations:** We may disclose your information where we are legally required to do so in order to comply with applicable law, governmental requests, a judicial proceeding, court order, or legal process, such as in response to a court order or a subpoena (including in response to public authorities to meet national security or law enforcement requirements).
+
+- **Vital Interests:** We may disclose your information where we believe it is necessary to investigate, prevent, or take action regarding potential violations of our policies, suspected fraud, situations involving potential threats to the safety of any person and illegal activities, or as evidence in litigation in which we are involved.
+
+More specifically, we may need to process your data or share your personal information in the following situations:
+
+- **Vendors, Consultants and Other Third-Party Service Providers.** We may share your data with third party vendors, service providers, contractors or agents who perform services for us or on our behalf and require access to such information to do that work. Examples include: payment processing, data analysis, email delivery, hosting services, customer service and marketing efforts. We may allow selected third parties to use tracking technology on the Sites or Apps, which will enable them to collect data about how you interact with the Sites or Apps over time. This information may be used to, among other things, analyze and track data, determine the popularity of certain content and better understand online activity. Unless described in this Policy, we do not share, sell, rent or trade any of your information with third parties for their promotional purposes.
```

Comment:
`hosting services, customer service, and marketing efforts.`    comma after service